test/Tests/src/Data/Time/Date_Spec.enso

```diff
@@ -229,6 +229,22 @@ spec_with name create_new_date parse_date =
             (create_new_date 2000 7 1).end_of Date_Period.Quarter . should_equal (Date.new 2000 9 30)
             (create_new_date 2000 6 30).end_of Date_Period.Quarter . should_equal (Date.new 2000 6 30)
 
+        Test.specify "should allow to compute the number of days until a date" <|
```

Comment:
Should we also add tests for the `next` method?

Reply:
yes I want to add tests for `next`, `previous` and `until`.